frontend/src/v5/ui/themes/overrides-mui.d.ts

```diff
@@ -33,6 +33,8 @@ interface IShadow {
 	level_8: string;
 	level_9: string;
 	level_10: string;
+	medium: string;
```

Comment:
There are 10 levels of shadow, we don't have `medium` or `large`. If there is something like that in the design, you have to ask George which level it suits.